tutorial/errors.po

```diff
@@ -148,7 +148,7 @@ msgstr ""
 "l'utilisateur d'interrompre le programme (en utilisant :kbd:`Control-C` ou "
 "un autre raccourci que le système accepte) ; notez qu'une interruption "
```

Comment:
```suggestion
"un autre raccourci que le système accepte) ; notez qu'une interruption "
```

/tmp/padpo_zivx6y2p/tutorial/errors.po: 136 [     NBSP     ] There should be a non-breakable space before ";": between ###courci que le système accepte) ### and ###; notez qu'une interruption gén###